Snippets/SqlServer/SqlServer_3.1/MultiCatalog.cs

```diff
@@ -0,0 +1,37 @@
+﻿using NServiceBus;
+using NServiceBus.Transport.SQLServer;
+
+class MultiCatalog
+{
+    void CatalogForQueue(EndpointConfiguration endpointConfiguration, IMessageSession messageSession)
+    {
+        #region sqlserver-multicatalog-config-for-queue
+
+        var transport = endpointConfiguration.UseTransport<SqlServerTransport>();
+
+        transport.UseCatalogForQueue(queueName: "myqueue", catalog: "MyCatalog");
+        transport.UseCatalogForQueue(queueName: "error", catalog: "ServiceControl");
+
+        messageSession.Send("myqueue", new MyMessage());
```

Comment:
It seems odd that this is right after configuring the transport. Maybe put ellipsis between those two sections to indicate they are not connected to each other or split sending into separate snippet altogether.